frontend/components/top_nav/SiteTopNav/SiteTopNav.tests.tsx

```diff
@@ -0,0 +1,351 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { createCustomRenderer } from "test/test-utils";
+
+import { noop } from "lodash";
+
+import createMockUser from "__mocks__/userMock";
+import createMockConfig from "__mocks__/configMock";
+import createMockTeam from "__mocks__/teamMock";
+
+import SiteTopNav from ".";
+
+const urlLocation = {
+  pathname: "queries/manage",
+  search: "",
+  query: {},
+};
+
+describe("SiteTopNav - component", () => {
```

Comment:
To be clear, these do not replace the "Log in as..." tests in QA wolf right? The screenshot checkmarks are a little ambiguous (e.g., is that a check mark over "Log in as Global Observer" or a double checkmark for the following test? I assume the latter?)